app/controllers/tasks_controller.rb

```diff
@@ -229,7 +229,8 @@ def tempfile_from_string(string)
   end
 
   def render_export_actions(task:, exported:, error: nil, task_found: nil, update_right: nil)
-    render_to_string(partial: 'export_actions.html.slim',
+    render_to_string(partial: 'export_actions',
+                     formats: [:html],
```

Comment:
The array is not mandatory here, is it? I am unsure what I prefer...

Either `formats: :html` or the version you chose `formats: [:html]`